src/components/Layout/GridTableLayout/QueryTable.tsx

```diff
@@ -0,0 +1,77 @@
+import { useMemo } from "react";
```

Comment:
Copied from https://github.com/homebound-team/internal-frontend/blob/main/src/components/QueryTable.tsx
with only a few type changes to avoid actual apollo client dependencies or importing `PageInfo` GQL type

Reply:
Oh nice! I forgot `QueryTable` hadn't made it into Beam yet :+1: